controllers/propagator/placementRuleMapper.go

```diff
@@ -34,8 +37,15 @@ func placementRuleMapper(c client.Client) handler.MapFunc {
 				subjects := pb.Subjects
 				for _, subject := range subjects {
 					if subject.APIGroup == policiesv1.SchemeGroupVersion.Group && subject.Kind == policiesv1.Kind {
-						log.Info("Found reconciliation request from placement rule...", "Namespace", object.GetNamespace(),
-							"Name", object.GetName(), "Policy-Name", subject.Name)
+						log.Info(
+							"Found reconciliation request from placement rule...",
+							"Namespace",
+							object.GetNamespace(),
+							"Name",
+							object.GetName(),
+							"Policy-Name",
+							subject.Name,
+						)
```

Comment:
Is it possible to combine keys and values on the same line for structured logging? Like:
```suggestion
						log.Info(
							"Found reconciliation request from placement rule...",
							"Namespace", object.GetNamespace(),
							"Name", object.GetName(),
							"Policy-Name", subject.Name,
						)
```
Or does one of the linter rules not like that?

Reply:
So this is just a matter of preference for me when the log line is too long since I find it easier to read since I know that each new line is a separate parameter. This is the approach taken by the black Python auto-formatting tool. I was just carrying that over to my Go coding.